internal/auth/handshake.go

```diff
@@ -129,10 +130,14 @@ func Handshake(cfg HandshakeConfig) error {
 			var msg message
 			err := wsConn.ReadJSON(&msg)
 			if err != nil {
-				if err == websocket.ErrReadLimit {
-					logs.Errorf("Websocket message too large for buffer - please increase buffer size")
+				switch {
+				case websocket.IsCloseError(err, websocket.CloseGoingAway):
```

Comment:
Why limit ourselves to `CloseGoingAway` error only? This works now if I shut down n8n gracefully (e.g. with SIGTERM), but if I kill it with `SIGKILL` (`kill -9 <pid>`), it exits the launcher:

```
2024/11/26 22:05:23 ERROR Failed to execute `launch` command: handshake failed: failed to read ws message: websocket: close 1006 (abnormal closure): unexpected EOF
```

Could we retry in all cases where the websocket disconnects?